src/centos-stream/manifest.json

```
@@ -0,0 +1,72 @@
+{
+  "repos": [
+    {
+      "name": "dotnet-buildtools/prereqs",
+      "images": [
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/centos-stream/9.0/amd64",
+              "os": "linux",
+              "osVersion": "centos-stream9",
+              "tags": {
+                "centos-stream9": {}
+              }
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/centos-stream/9.0/helix/amd64",
+              "os": "linux",
+              "osVersion": "centos-stream9",
+              "tags": {
+                "centos-stream9-helix": {}
+              }
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/centos-stream/9.0/mlnet/helix",
+              "os": "linux",
+              "osVersion": "centos-stream9",
+              "tags": {
+                "centos-stream9-mlnet-helix": {}
+              }
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/centos-stream/10.0/helix/",
+              "os": "linux",
+              "osVersion": "centos-stream10",
+              "tags": {
+                "centos-stream9-helix": {}
```

Two things here.

First is that this should be 10, not 9. This also needs to be applied to the arm64 Dockerfile. 

Second is to add a separator for the version to follow the existing pattern of other distros. This will actually be necessary if we want to use Renovate to automatically update tag references; it needs a common pattern to recognize. Since you're in here, can you apply this pattern to the version 9 tags as well (by adding a new tag, not replacing the existing one)?

```suggestion
                "centos-stream-10-helix": {}
```

---

I would be happy to make that change. I think that's a good direction.